### ISSUE TITLE
3.5 Platform Protocol - allow materials during challenge?

### ISSUE THREAD

**Issue by jswingle** [gryphonshafer/Quizzing-Rule-Book#125]
`Coaches may have quiz material(s) on the platform during a timeout or prior to a quiz.
`

What if we allowed quizzers to consult the rulebook during a challenge? This would potentially increase the strength of challenges, and it evens the playing field between the quizzers and the officials in a challenge (we don't expect the officials to know all the detailed rule language off the top of their head to make good decisions; why expect the quizzers to?)

We could also allow the material itself to be consultable during a challenge, but the advantage to this is not as obvious since we're encouraging them to memorize the material anyways. I don't think there would be a downside to it, frankly; it's not like the quizzers are going to be cheating on the questions by looking at the material, the question/answer period is already over by the time there's a challenge.

Such a rule change could be placed here in 3.5, or could be in the challenge section of the rulebook.

**Comment by jswingle** [comment]
@ARMediting since you assigned this issue to yourself, do you have any thoughts to add? It's not a priority issue but we should probably progress it at some point.

**Comment by Gpoulsen3hills** [comment]
If allowing the quizzers to consult the rule book or quiz material prior to a challenge that would exponentially increase the length of the whole procedure.

**Comment by josiah-leinbach** [comment]
I second what @Gpoulsen3hills has said. Challenges are already amongst the most lengthy procedures in quizzing and this only adds to their length.

**Comment by gryphonshafer** [comment]
There's nothing stopping a quizzer from launching a 17-minute filibuster challenge (while still being prohibited from referencing the rule book or material) except the quizmaster intervening.

Since the point of a challenge is to help being about more accurate rulings, the inverse of which is anti-mission, I feel that if a quizzer feels they'd provide a better challenge if they can consult the rule book or material, I don't want to block from them that option.

**Comment by jswingle** [comment]
> There's nothing stopping a quizzer from launching a 17-minute filibuster challenge... except the quizmaster intervening

Yes, and the rulebook also does not prescribe how the quizmaster can intervene in that situation, therefore we aren't breaking the rules if we throw the quizzer out the window for filibustering in this manner. =)

**Comment by levikoral** [comment]
I have sat through challenges, and of course they are lengthy. But I have seen quizzers who challenged, and even myself, not know exactly where something is located in the text. To clear that up the quizzers of course could look in Scripture itself to search for the verse. That would make the process take even longer.
So to avoid this, do not let the quizzers examine the text during or before a challenge. The only downfall is that a QM might not know exactly what a person is taking about. But the QMs should have enough materials there to be able to find the right solution.
